frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import { API_BASE_URL } from "@/config";
+
+import { Footer as IFooter, Logo as ILogo } from "@types/Footer";
+
+export const Footer: React.FC<IFooter> = ({disclaimer, logos, privacy}) => {
+    return (
+        <div className="aha__footer">
+            <div className="disclaimer" dangerouslySetInnerHTML={{
+                __html: disclaimer,
+                }}
+            />
+            <div className="logos">
+                {logos.map((logo: ILogo, index: number) => (
+                    <a
+                        href={logo.href}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        key={index}
+                    >
+                        <img src={API_BASE_URL + logo.file} alt={logo.alt} />
+                    </a>
+                ))};
+            </div>
+            <div className="disclaimer" dangerouslySetInnerHTML={{
```

Comment:
I see toontje hoger originally used to have a designated `.privacy` css class. Did you not use that class consciously?

Reply:
No, this is indeed a copy & paste error. Good catch!